src/OneBlinkFormBase.tsx

```diff
@@ -532,21 +550,33 @@ function OneBlinkFormBase({
     [definition],
   )
 
+  const addCaptchaRef = React.useCallback(
```

Comment:
This function should return a function to deregister the `ReCAPTCHA` so that the `useEffect()`s further down the tree can remove them if they get unmounted for any reason.